src/app/shared/components/login-buttons/login-buttons.component.html

```diff
@@ -1,9 +1,9 @@
 <div class="login-buttons-authorization">
   <button id="button-login" class="user-login-btn">
     <i class="fa fa-user" aria-hidden="true"></i>
-    User login
+    {{ this.authenticationService.getUserInfo()?.email }}
```

Comment:
getUserInfo uses local storage api. Accessing it frequently is performance inefficient. Just FYI.
In module 8 you will use observable here anyway. SO up to you if you want to optimize here